components/style/values/generics/grid.rs

```diff
@@ -701,3 +701,13 @@ pub enum GridTemplateComponent<L> {
     /// A `subgrid <line-name-list>?`
     Subgrid(LineNameList),
 }
+
+impl<L> GridTemplateComponent<L> {
+    /// Returns length of <track-size> list.
```

Comment:
https://dxr.mozilla.org/servo/rev/31228c18499d1c7f68b6b64b559354c768e81215/components/style/values/generics/grid.rs#209 ? Sorry, I didn't understand what you are saying in that comment actually.